modules/install/k3s_master.sh

```diff
@@ -195,16 +242,17 @@ main() {
   disable_cloud_setup
   install
   if [ "$etcd_only_node" -eq 0 ]; then
-    # If etcd only node count is 0, then wait for nodes/pods to come up. 
-    # etcd only node needs api server to come up fully, which is in control plane node. 
-    # and hence we cannot wait for node/pod status in this case. 
+    # If etcd only node count is 0, then wait for nodes/pods to come up.
+    # etcd only node needs api server to come up fully, which is in control plane node.
+    # and hence we cannot wait for node/pod status in this case.
     wait_nodes
     wait_ready_nodes
     wait_pods
   else
     # add sleep to make sure install finished and the node token file is present on the node for a copy
     sleep 30
   fi
+  install_etcdctl
```

Comment:
For k3s, we should also only install etcdctl when using an etcd DB. So when externaldb is passed, we should not run this at all (in addition to the other comments made on one of the rke2 files).

Reply:
done